build-tests/api-extractor-scenarios/etc/includeForgottenExports/api-extractor-scenarios.api.json

```diff
@@ -0,0 +1,638 @@
+            {
+              "kind": "Reference",
+              "text": "ForgottenExport1",
+              "canonicalReference": "api-extractor-scenarios!~ForgottenExport1:class"
+            },
+            {
+              "kind": "Content",
+              "text": ";"
+            }
+          ],
+          "returnTypeTokenRange": {
+            "startIndex": 1,
+            "endIndex": 2
+          },
+          "releaseTag": "Public",
+          "overloadIndex": 1,
+          "parameters": [],
+          "name": "someFunction1"
+        },
+        {
+          "kind": "Function",
+          "canonicalReference": "api-extractor-scenarios!someFunction2:function(1)",
+          "docComment": "/**\n * @public\n */\n",
+          "excerptTokens": [
+            {
+              "kind": "Content",
+              "text": "export declare function someFunction2(): "
+            },
+            {
+              "kind": "Reference",
+              "text": "DuplicateName",
+              "canonicalReference": "api-extractor-scenarios!~DuplicateName:type"
+            },
+            {
+              "kind": "Content",
+              "text": ";"
+            }
+          ],
+          "returnTypeTokenRange": {
+            "startIndex": 1,
+            "endIndex": 2
+          },
+          "releaseTag": "Public",
+          "overloadIndex": 1,
+          "parameters": [],
+          "name": "someFunction2"
+        },
+        {
+          "kind": "Function",
+          "canonicalReference": "api-extractor-scenarios!someFunction4:function(1)",
+          "docComment": "/**\n * @public\n */\n",
+          "excerptTokens": [
+            {
+              "kind": "Content",
+              "text": "export declare function someFunction4(): "
+            },
+            {
+              "kind": "Reference",
+              "text": "ForgottenExport4.ForgottenExport5",
+              "canonicalReference": "api-extractor-scenarios!~ForgottenExport4.ForgottenExport5:class"
+            },
+            {
+              "kind": "Content",
+              "text": ";"
+            }
+          ],
+          "returnTypeTokenRange": {
+            "startIndex": 1,
+            "endIndex": 2
+          },
+          "releaseTag": "Public",
+          "overloadIndex": 1,
+          "parameters": [],
+          "name": "someFunction4"
+        },
+        {
+          "kind": "Function",
+          "canonicalReference": "api-extractor-scenarios!someFunction5:function(1)",
+          "docComment": "/**\n * @public\n */\n",
+          "excerptTokens": [
+            {
+              "kind": "Content",
+              "text": "export declare function someFunction5(): "
+            },
+            {
+              "kind": "Content",
+              "text": "internal2."
+            },
+            {
+              "kind": "Reference",
+              "text": "ForgottenExport6",
+              "canonicalReference": "api-extractor-scenarios!ForgottenExport6:class"
```

Comment:
Observe that the canonical reference for `ForgottenExport6` as determined from the `.api.json` above is `api-extractor-scenarios!internal2.ForgottenExport6:class`. This is an existing bug with `AstNamespaceImport`s, discussed a bit in https://github.com/microsoft/rushstack/pull/3584#discussion_r944523487.